slackbridge/bots.py

```diff
@@ -14,6 +18,60 @@ class IRCBot(irc.IRCClient):
 
 class BridgeBot(IRCBot):
 
+    @functools.total_ordering
+    class SlackMessage:
+        def __init__(self, raw_message, user_bots, channels):
```

Comment:
Ideally I think this would not have to take in `user_bots` or `channels`, but I'm not sure what the best way around this would be. Maybe pass in a reference to the `BridgeBot` instance so that the message can access the bots and channels from there?